source/GameInterface/Services/Equipments/Patches/EquipmentLifetimePatches.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using GameInterface.Policies;
+using HarmonyLib;
+using Serilog;
+using System;
+using TaleWorlds.Core;
+using GameInterface.Services.Equipments.Messages.Events;
+
+
+namespace GameInterface.Services.Equipments.Patches;
+
+/// <summary>
+/// Patches for lifecycle of <see cref="Equipment"/> objects.
+/// </summary>
+[HarmonyPatch]
+internal class EquipmentLifetimePatches
+
+    
+{
+    private static readonly ILogger Logger = LogManager.GetLogger<EquipmentLifetimePatches>();
+    private static bool skip = false;
+
+    [HarmonyPatch(typeof(Equipment), MethodType.Constructor)]
+    [HarmonyPrefix]
+    private static bool CreateEquipmentPrefix(Equipment __instance)
+    {
+        // Skip chained parameterless constr after Equipment(bool isCivilian) constr
+        if (skip) { 
```

Comment:
Can we not create the equipment without using the constructor and let sync handle those things instead?

ObjectHelper.SkipConstructor< Equipment >()

Reply:
Actually, skip this. I haven't looked at equipment enough. If this works, I'm fine with it.